common/source/docs/common-auxiliary-functions.rst

```diff
@@ -374,7 +376,8 @@ Other functions are:
                                          | For developers only.
     AttCon Accel Limits                  | Turns on/off attitude controller acceleration limits.
                                          | For developers only.
-    Retract Mount                         Move the :ref:`camera mount <common-cameras-and-gimbals>` to its retracted position.
+    Retract Mount1                        Move the :ref:`camera mount <common-cameras-and-gimbals>` to its retracted position.
+    Retract Mount2                        Move the :ref:`camera mount <common-cameras-and-gimbals>` to its retracted position.
```

Comment:
move retract mnt2mnt2 to between SwitchExternalAHRS     and   ARM/DISARM (4.2 and higher)